grpc/interceptors.go

```diff
@@ -7,20 +7,38 @@ import (
 	"google.golang.org/grpc"
 )
 
-// defaultMethodTimeout is the default context timeout for all inbound gRPC
-// methods used when no deadline is set on the context.
-var defaultMethodTimeout = 10 * time.Minute
+// DefaultMethodTimeout is the default context timeout for all inbound and outbound 
+// gRPC methods used when no deadline is set on the context.
```

Comment:
```suggestion
// DefaultMethodTimeout is the default context timeout for all inbound gRPC
// and outbound gRPC methods to modules, only used when no deadline is 
// set on the context.
```
[nit] Technically not _all_ outbound gRPC methods. In particular, outbound messages to app through the [RobotService](https://github.com/viamrobotics/api/blob/main/proto/viam/app/v1/robot.proto#L13), for example, have no default timeout that I'm aware of.

Reply:
ah yeah good suggestion, that is more accurate